NativeApp/src/components/Login/Form/index.js

```diff
@@ -38,45 +38,68 @@ class LoginForm extends Component {
     const { handleLogin, hasError, loading } = this.props;
     const { changeColor } = this;
     return (
-      <View>
-        <TextInput
-          id="input-1"
-          onFocus={() => changeColor('input-1', '#00DCFA')}
-          onBlur={() => changeColor('input-1', 'gray')}
-          placeholder="Email"
-          onChangeText={text => this.setState({ email: text })}
-          value={email}
-          autoCapitalize="none"
-          underlineColorAndroid="transparent"
-          selectionColor="#00DCFA"
-          style={{ borderBottomWidth: 1, borderColor: hasError ? 'red' : underlineColor1 }}
-        />
-        <TextInput
-          id="input-2"
-          onFocus={() => changeColor('input-2', '#00DCFA')}
-          onBlur={() => changeColor('input-2', 'gray')}
-          placeholder="Password"
-          onChangeText={text => this.setState({ password: text })}
-          value={password}
-          secureTextEntry
-          underlineColorAndroid="transparent"
-          style={{ borderBottomWidth: 1, borderColor: hasError ? 'red' : underlineColor2 }}
-          selectionColor="#00DCFA"
-        />
-        {
-          hasError && (
-            <Caption type="base" style={styles.red}>
-              Incorrect email or password
-            </Caption>
-          )
-        }
+      <View style={styles.view}>
+        <View>
+          <FormLabel>Email</FormLabel>
+          <FormInput
+            id="input-1"
+            onChangeText={text => this.setState({ email: text })}
+            inputStyle={{
+              fontSize: 20,
+              color: 'black',
+              padding: 8,
+              borderColor: hasError ? 'red' : underlineColor1,
+              borderBottomWidth: 1,
+              width: '100%',
+            }}
+            underlineColorAndroid="transparent"
+            selectionColor="#00DCFA"
+            placeholder="you@email.com"
+            autoCapitalize="none"
+            value={email}
+            returnKeyType="next"
+            onSubmitEditing={() => this.secondInput.focus()}
+            onFocus={() => changeColor('input-1', '#00DCFA')}
+            onBlur={() => changeColor('input-1', 'gray')}
+            blurOnSubmit={false}
+          />
+          <FormLabel>Password</FormLabel>
+          <FormInput
```

Comment:
This file is getting a bit hard to read. I think we should move `FormInput` into it's own component. A lot of duplicate code that could be squashed down into one file. We could also move the `changeColor` method to the same file and rework it slightly to use `ref` instead of passing our own id.

Reply:
i have made formInput into its own component. please could you check this and feel free to make any suggestions :)